docs/release-notes.rst

```diff
@@ -1,3 +1,199 @@
+
+**Workaround**: Uninstall and re-install the application to clean out the F5 ACI ServiceCenter database.
+
+------
+
+
+L4-L7 App Services
+------------------
+
+**Application services declaration not deleted**
+
+If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, the Application Services configuration may not be deleted completely, even after multiple attempts from the application UI. However, the configuration gets removed from the BIG-IP device.
+
+**Workaround**: Upload one more AS3 sample declaration to the app and then perform a :guilabel:`Delete all` operation. (Use :guilabel:`View AS3 Declaration` and click :guilabel:`Delete`.)
+
+-------
+
+
+Dynamic Endpoint Attach Detach
+------------------------------
+
+**Dynamic Endpoint attach/detach is not supported for BIG-IP High Availability setups**
+
+Dynamic Endpoint attach/detach using the "Manage Endpoint Mappings" button is not supported for BIG-IP devices which are in HA pair. If used, the behavior is unknown and user may experience BIG-IP service restarts and hang up.
+
+**Workaround:** None
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/238
+
+------
+
+**Error on EPG mapping delete operation**
+
+When a dynamic endpoint mapping is added to an application using **Manage Endpoint Mappings**, the application gets created on the BIG-IP. If this mapping is deleted using the **RESET** button on **Manage Endpoint Mappings** form, user may encounter an error "The requested Pool Member (/Partition/App/Pool /NodePartition/NodeIP) was not found"
+
+**Workaround:** Click the **Submit** button again, and the mapping will be reset properly without any errors. 
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/185
+ 
+------
+
+**AS3 application can either have static nodes or dynamic nodes but not both**
+
+AS3 applications will support either static nodes or dynamic nodes (using the **Manage Endpoint Mappings** button) but not both. 
+
+------
+
+**Dynamic endpoints will not be discovered if any of the dynamic endpoint IPs already belong to the static nodes on the BIG-IP**
+
+Dynamic endpoints are the endpoints present in APIC Endpoint Group. The app automatically updates this in the BIG-IP application’s pool members provided the correct association is configured via the application. But if this dynamic endpoint list consists of an IP which has already been added as a static node on the BIG-IP previously, none of the dynamic endpoints will get updated in the application. 
```

Comment:
But if this dynamic endpoint list consists of an IP which has already been added as a static node on the BIG-IP, ...

(don't need 'already' and then 'previously' in that sentence.. you could change it to ".. which has previously been added" but don't need it in both spots.